### DISCUSSION TITLE
Make Mower dock when raining

### DISCUSSION THREAD

**Discussion by Din-BH** [Thomas55555/husqvarna_automower#466] [Q&A] [answered]
Hi all! I want to make my Automower 405x dock for some time when raining. In itself that is not so difficult, but I want to avoid messing with situations where the Mower actions have been overridden manually. Essentially this means that the Mower either has been put in "Park until further notice" mode or "Paused" mode. The second one of these is easy to find, but does anyone have a solid way to find out if the mower has been parked until further notice?

Next step after I manage this and find out what is the "right" amount of time to wait after rain (or if there is any point in this at all), will be to still make sure I get something done when the rain just keeps on drizzling.

If you have some good knowledge or existing setups, please share!

**Comment by Thomas55555** [comment]
In this case, the mower is parked until further notice.
https://github.com/Thomas55555/husqvarna_automower/blob/7fa28854c9b773f570d945ed24b4a9a31fbb30cf/custom_components/husqvarna_automower/vacuum.py#LL204C1-L214C53
I think it's not possible for the user to identify this state. I could make a switch for that or extend the problem sensor with this functionallity.

**Comment by Thomas55555** [comment, marked as answer]
You can test version 2023.5.1b8.
There it is in the problem sensor.

**Reply by Din-BH** [reply]
Thanks! I'll check it out

**Comment by 0ldF0x** [comment]
I'm using "[Optical Rain Sensor RG-9](https://rainsensors.com/products/rg-9/)" + automation with 1min park-timer, works perfect.
